src/OrbitGl/TimeGraph.cpp

```diff
@@ -592,13 +592,13 @@ void TimeGraph::ProcessPagefaultTrackingTimer(const orbit_client_protos::TimerIn
         absl::StrFormat("Process [%s]", capture_data_->process_name()),
         absl::StrFormat("CGroup [%s]", cgroup_name), "System"};
     track = track_manager_->CreateAndGetPagefaultTrack(kSeriesNames);
+    track->SetIndexOfSeriesToHighlightForMajorPagefaultTrack(0);
```

Comment:
Couldn't this be set only once?

Reply:
Or why not never? Like, by default in the creation could be 0?

Reply:
We do set it only once, i.e., when we create the pagefault track.

The reason that I don't set this inside `CreateAndGetPagefaultTrack` is, we have the series name here, so we know we are highlighting which series.

Reply:
If we set 0 as default, that means we also do the highlighting for the minor pagefault.